main.go

```diff
@@ -34,6 +34,9 @@ func main() {
 	http.HandleFunc("/get", GetCommentsHandler)
 
 	port := os.Getenv("COMMENTO_PORT")
+	if port == "" {
+		port = "8080"
+	}
```

Comment:
This is already done: see [config.go:12](https://github.com/adtac/commento/blob/master/config.go#L12).

Reply:
Ah yes, that's right. For some reason it didn't work at some point. I should've checked `config.go`. This will be resolved.